DenunciaCiudadana/src/components/Complaint.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {View, Text, Image, TouchableOpacity} from 'react-native';
+import {stylesHome} from '../assets/styles/styleHomeDesing';
+
+const Complaint = ({item}) => {
+  return (
+    <View style={stylesHome.complaint}>
+      <View>
+        <Text style={stylesHome.titleComplaint}>Nos cortaron el internet</Text>
```

Comment:
Try to move all the text into the text constant file, it helps you with the i18n.

```suggestion
        <Text style={stylesHome.titleComplaint}>{texts.someoneDidNotPayBill}</Text>
```

Reply:
It is just for the design, in the next commit I did the change, where I got the info for firebase